pkgs/tools/graphics/vulkan-tools/default.nix

```diff
@@ -61,14 +61,11 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = lib.optionalString stdenv.isDarwin ''
-    # Modify mac_common.cmake to find the ICD where nixpkgs puts it.
-    substituteInPlace mac_common.cmake \
-      --replace MoltenVK/icd/MoltenVK_icd.json MoltenVK_icd.json
+    # Modify cube/CMakeLists.txt to find the ICD where nixpkgs puts it.
+    substituteInPlace cube/CMakeLists.txt \
+      --replace-fail MoltenVK/icd/MoltenVK_icd.json MoltenVK_icd.json
```

Comment:
If the ICD is only relevant for the cube demo now, and the cube demo is not being built, then it seems like this can be dropped completely.

Reply:
Dropped. Thanks!

Reply:
The cube demo requires `ibtool`, so it still needs to be disabled. There’s just no need to fix up its `CMakeFiles.txt` to have the correct ICD path since it won’t ever be built.

Reply:
(Unless it’s no longer built by default, then dropping `postPatch` is okay.)

Reply:
It’s not built, so this is resolved and can be disregarded.